src/main/resources/application.conf

```diff
@@ -0,0 +1,7 @@
+ktor {
+  environment = local
+  environment = ${?KTOR_ENV}
```

Comment:
Overskriver med verdi fra nais-filene når man ikke er lokalt.

Reply:
Hmja, vi bruker den for å sette environment i andre apper, men kan hende man kunne løst det på en annen måte :thinking: